Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -1159,7 +1158,7 @@ end
     loadp (constexpr (JSWebAssemblyInstance::offsetOfVM()))[wasmInstance], a0
     copyCalleeSavesToVMEntryFrameCalleeSavesBuffer(a0, a1)
 
-if ASSERT_ENABLED
+if (not USE_BUILTIN_FRAME_ADDRESS) or ASSERT_ENABLED
```

Comment:
I think that we should just make ARMv7 work with `BUILTIN_FRAME_ADDRESS`. It is the last JIT architecture which is not supporting this `BUILTIN_FRAME_ADDRES`.

Reply:
In IPInt side, I think we should just put this frame without ASSERT_ENABLED. We would like to support arbitrary arch in IPInt, and we may support things without USE_BUILTIN_FRAME_ADDRESS. But anyway, I think definitely all JIT arch should support USE_BUILTIN_FRAME_ADDRESS

Reply:
But for this particular case, I think we should just enable USE_BUILTIN_FRAME_ADDRESS for ARMv7. That's the simplest solution.

Reply:
Agreed. However it looks like we get broken code when doing a Release build with GCC on ARMv7. I can see it doing

```
sub     sp, #180        @ 0xb4
add     r7, sp, #24
```

and then dereferencing that to get the caller frame address. `clang` correctly sets up `r7` before allocating stack space. Looking for a workaround.

Reply:
I don't see any way to work around this, I think this needs an upstream GCC fix. How about we gate this on `ARMv7` in WebAssembly.asm for now?